modules/oe_whitelabel_paragraphs/tests/src/Functional/GalleryParagraphTest.php

```diff
@@ -206,7 +206,7 @@ protected function assertParagraphRendering(array $expected, ParagraphInterface
    *
    * @throws \Exception
    */
-  protected function renderParagraph(ParagraphInterface $paragraph, string $langcode = NULL): string {
+  protected function renderParagraph(ParagraphInterface $paragraph, ?string $langcode = NULL): string {
```

Comment:
Param description doesn't seem to match: Rendering language code, defaults to 'en'.